src/main/java/org/tdl/vireo/controller/SubmissionController.java

```diff
@@ -990,9 +990,18 @@ public ApiResponse removeFile(@WeaverUser User user, @PathVariable Long submissi
             apiResponse = new ApiResponse(ERROR, "You are not allowed to delete license files!");
         } else {
             if (user.getRole().equals(Role.ROLE_ADMIN) || user.getRole().equals(Role.ROLE_MANAGER) || uri.contains(String.valueOf(hash))) {
-                JsonNode fileInfo = assetService.getAssetFileInfo(uri, submission);
+                String fileName = "";
+                String fileSize = "file not found";
+                if (assetService.assetFileExists(uri)) {
+                    JsonNode fileInfo = assetService.getAssetFileInfo(uri, submission);
+                    fileName = fileInfo.get("name").asText();
+                    fileSize = fileInfo.get("readableSize").asText();
+                } else {
+                    fileName = assetService.getAssetFileName(uri);
```

Comment:
Why present a filename to the front end at all? Who deleted the file if the user was unaware of it?